software/building-block-view.qmd

```diff
@@ -111,19 +113,6 @@ interface.](/design/images/component-ui.png)
     been granted access to as well as simple aggregations of the
     variables.
 
-### User Authentication (white box)
-
-<!-- TODO: Need a components diagram for this. -->
-
-![user id](/design/images/user-id.png)
-
-| Building Block           | Description                                   |
-|--------------------------|-----------------------------------------------|
-| Connect Remote ID Server | Connect to remote user authentication server  |
-| Local Authentication     | Use Seedcase local user authentication server |
-
-: Black boxes within the *User Authentication* part.
-
```

Comment:
This is the only thing that I removed completely because the diagram didn't exists and for now we don't have a remote ID server